src/core/PWScore.cpp

```diff
@@ -1309,9 +1309,11 @@ int PWScore::ReadFile(const StringX &a_filename, const StringX &a_passkey,
   std::sort(m_InitialEmptyGroups.begin(), m_InitialEmptyGroups.end());
 
   if (pRpt != nullptr) {
-    std::wstring cs_title;
-    LoadAString(cs_title, IDSC_RPTVALIDATE);
+#if !defined(_WIN32) || defined(__WX__)
```

Comment:
There's no good reason for different behavior between Windows and wx here.
I'll unify this post-merge.

Reply:
Reason for this difference is, that language depending translation of the action is done at this central place. In WX environment the calling function is handing over the not translated string of the action. This result in to unique English file names and  simplified supporting functions. Please rethink on the proposed change.

Reply:
I don't see why Windows should behave differently, especially since there's conditional compilation to keep this different.

Reply:
Ok for me to drive windows in the direction of wxWidgets